docs/configure.rst

```diff
@@ -150,6 +160,8 @@ The available partition attributes are the following:
 * ``environs``: A list of environments, with which ReFrame will try to run any regression tests written for this partition (default ``[]``).
   The environment names must be resolved inside the ``environments`` section of the ``site_configuration`` dictionary (see `Environments Configuration <#environments-configuration>`__ for more information).
 
+* ``container_platforms``: A dictionary with the container platforms supported in the partition and the corresponding modules to load them.
+
```

Comment:
Here you should expand a bit more. This is a dictionary describing the environment for setting up this container platform. You should also describe which are the supported container platforms and provide links to them.

Reply:
I fixed that.